### PR TITLE
🌐 [i18n-KO] Translated `package_reference/collections.md` to Korean

### PR DESCRIPTION
**Pull request by boyunJang**
# What does this PR do?

Translated the `package_reference/collections.md` file of the documentation to Korean.
Thank you in advance for your review. 

Part of https://github.com/huggingface/huggingface_hub/issues/1626

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [ ] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team Hugging Face KREW, may you please review this PR? 
@harheem, @cjfghk5697, @seoulsky-field, @nuatmochoi, @boyunJang, @SeungAhSon, @seoyoung-3060, @junejae, @usr-bin-ksh, @fabxoe

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

@jungnerd, @heuristicwave May you please review this PR?

### CODE REVIEW

**Comment by SeungAhSon** [comment]
LGTM!

**Comment by fabxoe** [comment]
LGTM!

File: docs/source/ko/package_reference/collections.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--⚠️ Note that this file is in Markdown but contain specific syntax for our doc-builder (similar to MDX) that may not be
+rendered properly in your Markdown viewer.
+-->
+
+# 컬렉션 관리[[managing-collections]]
+
+[`HfApi`] 문서 페이지에서 Hub에서 스페이스를 관리하는 메소드에 대한 참조를 확인하세요.
```
Comment (boyunJang) [inline docs/source/ko/package_reference/collections.md:7]:
```suggestion
Hub에서 Space를 관리하는 메소드에 대한 자세한 설명은 [`HfApi`] 페이지를 확인하세요.
```

**Comment by jungnerd** [comment]
@Wauplin LGTM! This PR is ready to merge!

**Review by Wauplin — APPROVED** [review]
Thanks @boyunJang for the docs and @SeungAhSon @fabxoe @jungnerd for the reviews!

**Pull request merged**